### PR TITLE
Post-Salad Types

### PR DESCRIPTION
**Pull request by danbills**
Schema salad de-sugars lots of things for us, making it easier for us to parse the CWL JSon.

Thanks Salad!

### CODE REVIEW

**Comment by geoffjentry** [comment]
This is pretty cool, even if slightly less sweet :) 

👍

File: cwl/src/main/scala/wdl4s/cwl/Cwl.scala [hunk 1]
```diff
@@ -15,9 +15,9 @@ sealed trait Cwl {
 case class Workflow(
   cwlVersion: Option[CwlVersion] = Option(CwlVersion.Version1),
   `class` : Workflow.`class`.type = Workflow.`class`,
-  inputs: WorkflowInput,
-  outputs: WorkflowOutput,
-  steps: WorkflowSteps) extends Cwl
+  inputs: Array[InputParameter] = Array.empty,
```
Comment (mcovarr) [inline cwl/src/main/scala/wdl4s/cwl/Cwl.scala]: I kinda liked having the `type`s, the `Coproduct`s not so much 🙂

**Comment by mcovarr** [comment]
:+1: grr forgot this repo uses the old steam-powered thumb approvals

**Comment by danbills** [comment]
(I'm fixing a couple broken tests)

**Comment by cjllanwarne** [comment]
@danbills prefer "squash then merge" to leave `oops left a println` out of develop history 😄

**Pull request merged**